src/main/scala/devices/debug/Periphery.scala

```diff
@@ -159,17 +159,18 @@ trait HasPeripheryDebugModuleImp extends LazyModuleImp {
     debug.map(_.disableDebug.foreach { x => dtm.io.jtag.TMS := sj.jtag.TMS | x })  // force TMS high when debug is disabled
     val psdio = psd.psd.getOrElse(WireInit(0.U.asTypeOf(new PSDTestMode)))
 
-    dtm.clock          := sj.jtag.TCK
+    dtm.io.jtag_clock  := sj.jtag.TCK
     dtm.io.jtag_reset  := sj.reset
     dtm.io.jtag_mfr_id := sj.mfr_id
     dtm.io.jtag_part_number := sj.part_number
     dtm.io.jtag_version := sj.version
-    dtm.reset          := dtm.io.fsmReset
+    dtm.io.tapReset     := dtm.io.fsmReset
```

Comment:
since we aren't actually using the module implicit reset and we can do `withClockAndReset`, do we actually need to bring this back out as an IO? Can we just keep this logic internnal to the DTM?

Reply:
Moved to internal logic.